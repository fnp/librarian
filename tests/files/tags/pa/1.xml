<akap>
  Bieży<pa><slowo_obce>bieży</slowo_obce> --- biegnie.</pa>.
</akap>
