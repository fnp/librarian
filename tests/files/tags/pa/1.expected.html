<div id="book-text">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Bieży<a href="#footnote-id1" class="annotation-anchor" id="anchor-id1">[1]</a>.</p>
<div id="footnotes">
<h3>Przypisy</h3>
<div class="fn-pa">
<a name="footnote-id1"></a><a href="#anchor-id1" class="annotation">[1]</a><p><em class="foreign-word">bieży</em> — biegnie. [przypis autorski]</p>
</div>
</div>
</div>
